packages/components/src/HopperProvider/tests/chromatic/HopperProvider.stories.tsx

```diff
@@ -23,36 +24,32 @@ const meta: Meta<typeof HopperProvider> = {
             disabled: true // This file is used to test the HopperProvider in isolation, so we don't want to render the HopperProvider here
         }
     } satisfies WithHopperStorybookAddonParameter
-};
+} satisfies Meta<typeof HopperProvider>;
```

Comment:
why was this changed? It's not using it in the storybook docs https://storybook.js.org/docs/writing-stories

Reply:
Make sure you change "TypeScript" to "Typescript 4.9" as the language in the documentation.

Reply:
wrote to you in slack with examples